lib/ocpp/v201/charge_point.cpp

```diff
@@ -174,9 +176,12 @@ ChargePoint::ChargePoint(const std::map<int32_t, int32_t>& evse_connector_struct
                                                            transaction_meter_value_callback, pause_charging_callback)));
     }
 
+    this->smart_charging_handler = std::make_shared<SmartChargingHandler>(this->evses, this->device_model);
+
     // configure logging
     this->configure_message_logging_format(message_log_path);
 
+    // TODO - this should this be dependency injected.
```

Comment:
Was this left here intentionally? If so, there's an extra `this`:
```suggestion
    // TODO - This should be dependency injected.
```

Reply:
Ah, that's from Coury's PR. Those commits will be dropped once https://github.com/EVerest/libocpp/pull/685 is merged.